package-lock.json

```diff
@@ -0,0 +1,3325 @@
+{
+  "name": "shardeum-website",
```

Comment:
@anvaynk had set up the project with **Yarn**. 
But if the majority of us use `npm` we can use that. ( btw I use **Yarn**) :) 
Let's settle on a common package manager. I'm okay with either of them.

Reply:
Sure I will also use yarn, and remove package lock

Reply:
Let's use Yarn only.

Reply:
Done